index.html

```diff
@@ -302,6 +302,12 @@
 		<td><code>0.1 + 0.2.</code></td>
 		<td>0.30000000000000004</td>
 	</tr>
+	<!-- Swift -->
+	<tr>
+		<td>Swift</td>
+		<td><code>0.1 + 0.2.</code></td>
```

Comment:
Is the period at the end here necessary? I ask because the other swift pr doesn't have one. https://github.com/erikwiffin/0.30000000000000004/pull/26